cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -12,124 +12,249 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 // Each packages v1alpha1 plugin must implement at least the following rpcs:
 //
 service PackagesService {
-  rpc GetAvailablePackages(kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetAvailablePackagesResponse) {
+  rpc GetAvailablePackageSummaries(GetAvailablePackageSummariesRequest) returns (GetAvailablePackageSummariesResponse) {
+     option (google.api.http) = {
+       get: "/core/packages/v1alpha1/availablepackagesummaries"
+     };
+   }
+  rpc GetAvailablePackageDetail(GetAvailablePackageDetailRequest) returns (GetAvailablePackageDetailResponse) {
     option (google.api.http) = {
-      get: "/core/packages/v1alpha1/packages"
+      get: "/core/packages/v1alpha1/availablepackagedetails"
     };
   }
-  rpc GetPackageRepositories(kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesRequest) returns (kubeappsapis.core.packages.v1alpha1.GetPackageRepositoriesResponse) {
+  rpc GetPackageRepositories(GetPackageRepositoriesRequest) returns (GetPackageRepositoriesResponse) {
```

Comment:
I'm keen to remove this from the core packages API (that is, any CRUD for repositories). It's no problem if a plugin defines it (we did add them to test things the other week), but when we finish the packages API, I expect we'll design and add a separate one for CRUD actions on repositories.

Reply:
Just confirming after having gone through the doc (and updating a little from some discussion), let's delete `GetPackageRepositories` from this file, as well as the `PackageRepository` message (which will no longer be needed, more below).